specification/fleet/search/SearchRequest.ts

```diff
@@ -52,8 +52,9 @@ import { Duration } from '@_types/Time'
 import { Checkpoint } from '../_types/Checkpoints'
 
 /**
- * The purpose of the fleet search api is to provide a search api where the search will only be executed
- * after provided checkpoint has been processed and is visible for searches inside of Elasticsearch.
+ * Run a Fleet search.
+ * The purpose of the Fleet search API is to provide a search api where the search will be run only
```

Comment:
```suggestion
 * The purpose of the Fleet search API is to provide a search API where the search will be run only
```

Reply:
actually maybe just "provide an API where..." ?

Reply:
Thanks! Updated in 2934aade66cf6f0ab83c78699de166ce675bc2db